scripts/ci/setup-travis.sh

```diff
@@ -58,25 +59,27 @@ if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
 fi
 
 if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
-    brew update
-    if [[ "$CC" == "gcc" ]]; then
-        brew install gcc@5
+    if [[ "$TRAVIS_COMPILER" == "gcc" ]]; then
         export CC=gcc-5
         export CXX=g++-5
+
+        # Filter out 'warning: section "__textcoal_nt" is deprecated'
+        # from the compiler output, as it makes the log reach the size limit
```

Comment:
Weird, but that could have been the issue.

Reply:
I think the main issue was with `set -e` at the top of this script. `rvm` (ruby version manager) was failing later on, and setting this made the whole build fail.

Reply:
Odd, but good to know!